### DISCUSSION TITLE
qBittorrent: Inject to category?

### DISCUSSION THREAD

**Discussion by SawkeeReemo** [cross-seed/cross-seed#252] [Q&A] [answered]
I use Sonarr and Radarr. When a torrent finishes being ingested, those apps change the qBT category to a "done" category, which also moves the files to a subfolder for file management.

Can cross-seed inject the torrents it makes to that "done" location? (which is different per app)

**Comment by Drazzilb08** [comment, marked as answer]
this is currently being discussed in #254 as this is an issue that might be able to be resolved going about this in a different manner.
